twint/tweet.py

```diff
@@ -158,6 +158,10 @@ def getRetweet(tw, _config):
 #     return t
 
 
+Tweet_formats = {
+    'datetime': '%d-%m-%Y %H:%M:%S %Z'
+}
+
```

Comment:
I think this is the better approach, we can use this to keep the date formatting consistent throughout the library and can add different formatting if required.
let me know what you think about this.